molmod/__init__.py

```diff
@@ -6,6 +6,7 @@
 from logging.config import dictConfig
 
 from flask import Flask
+from flask_cas import CAS
```

Comment:
`flask_cas` seems kind of abandoned - the sourcce repository (and indeed the user of that repository) no longer exists. :( [I'm uncertain about policies github have in place if that makes it suspectible to hijacking.]

If authentication uses the apereo implementation, it does at least support other IdP protocols (SAML, Oauth2/OpenID Connect). That might be worth exploring (if it's enabled/could be enabled). There also seems to be another module (python-cas) that could possibly be used if one will have acces to CAS only - https://djangocas.dev/blog/python-cas-flask-example/.

Reply:
Yes, was very confused by this situation :( There are several 'maintenance forks' for Flask-CAS (e.g.(https://github.com/MasterRoshan/flask-cas-ng), but the latest release of any Flask CAS extension still seems to come from this 'abandoned' repo (Flask-CAS==1.0.2), and it has a very handy decorator function that does all the work for me, so I used it. Mentioned this to @Norling earlier, who seemed to think it was OK to use, but perhaps you should discuss it with him. How would hijacking work?

CAS is what the Bioatlas uses, and I have limited knowledge of authentication. Also, login would only be used for restricting file upload and, probably, download (for user stats to VR), so will not give access to the database, for instance. Not sure if that makes it less risky...?